centreon/src/Centreon/Infrastructure/Monitoring/MonitoringRepositoryRDB.php

```diff
@@ -357,6 +357,11 @@ public function findHostGroups(?int $hostId): array
             'poller.id' => 'h.instance_id'
         ];
 
+        // To allow to find host groups relating to Service information
+        $ServiceConcordanceArray = [
+            'service.display_name' => 'srv.display_name',
+        ];
```

Comment:
PHP uses camelCase for varirable definitions
```suggestion
        $serviceConcordanceArray = [
            'service.display_name' => 'srv.display_name',
        ];
```